packages/TBC20/.gitignore

```diff
@@ -1,4 +1,3 @@
 /node_modules
 /dist
-/build
 *.js
```

Comment:
I think we should remove the *.js, because now we are uploading the build files, so token.js should be uploaded in this PR too.